.buildkite/pipeline.yml

```diff
@@ -647,11 +647,211 @@ steps:
+      - label: "Trivy Scan amd64 extensible Dockerfile image"
+        timeout_in_minutes: 10
+        depends_on:
+          - test_extensible_dockerfile_image_amd64
+        key: "trivy-scan-amd64-extensible-dockerfile-image"
+        agents:
+          provider: k8s
+          image: "docker.elastic.co/ci-agent-images/trivy:latest"
+        command: |-
+          mkdir -p .artifacts
+          buildkite-agent artifact download '.artifacts/*.tar.gz*' .artifacts/ --step build_extensible_dockerfile_image_amd64
+          trivy --version
+          env | grep TRIVY
+          find .artifacts -type f -name '*.tar.gz*' -exec trivy image --quiet --input {} \;
+
+      # ----
+      # Extensible Dockerfile.ftest build, tests and vunlerability scan on amd64
+      # ----
+      - label: "Building amd64 Docker image from extensible Dockerfile.ftest"
+        agents:
+          provider: aws
+          instanceType: m6i.xlarge
+          imagePrefix: ci-amazonlinux-2
+        env:
+          ARCHITECTURE: "amd64"
+          DOCKERFILE_PATH: "Dockerfile.ftest"
+          DOCKER_IMAGE_NAME: "docker.elastic.co/ci-agent-images/elastic-connectors-extensible-dockerfile-ftest"
+          DOCKER_ARTIFACT_KEY: "elastic-connectors-extensible-dockerfile-ftest"
+        command: ".buildkite/publish/build-docker.sh"
+        key: "build_extensible_dockerfile_ftest_image_amd64"
+        artifact_paths: ".artifacts/*.tar.gz"
+      - label: "Testing amd64 image built from extensible Dockerfile.ftest"
+        agents:
+          provider: aws
+          instanceType: m6i.xlarge
+          imagePrefix: ci-amazonlinux-2
+        env:
+          ARCHITECTURE: "amd64"
+          DOCKERFILE_PATH: "Dockerfile.ftest"
+          DOCKER_IMAGE_NAME: "docker.elastic.co/ci-agent-images/elastic-connectors-extensible-dockerfile-ftest"
+          DOCKER_ARTIFACT_KEY: "elastic-connectors-extensible-dockerfile-ftest"
+        depends_on: "build_extensible_dockerfile_ftest_image_amd64"
+        key: "test_extensible_dockerfile_ftest_image_amd64"
+        commands:
+          - "mkdir -p .artifacts"
+          - buildkite-agent artifact download '.artifacts/*.tar.gz*' .artifacts/ --step build_extensible_dockerfile_ftest_image_amd64
+          - ".buildkite/publish/test-docker.sh"
+      - label: "Trivy Scan amd64 extensible Dockerfile.ftest image"
+        timeout_in_minutes: 10
+        depends_on:
+          - test_extensible_dockerfile_ftest_image_amd64
+        key: "trivy-scan-amd64-extensible-dockerfile-ftest-image"
+        agents:
+          provider: k8s
+          image: "docker.elastic.co/ci-agent-images/trivy:latest"
+        command: |-
+          mkdir -p .artifacts
+          buildkite-agent artifact download '.artifacts/*.tar.gz*' .artifacts/ --step build_extensible_dockerfile_ftest_image_amd64
+          trivy --version
+          env | grep TRIVY
+          find .artifacts -type f -name '*.tar.gz*' -exec trivy image --quiet --input {} \;
```

Comment:
I think we are not going to get a lot of value from these steps, given that `Dockerfile.ftest` is almost equivalent to `Dockerfile`.

So I'm wondering: what if instead (maybe in a different PR, as a follow-up) we delete the two `.ftest` Dockerfiles and we try to live with only two Dockerfiles instead of four (`Dockerfile` and `Dockerfile.wolfi`), and we move the `RUN .venv/bin/pip install -r requirements/ftest.txt` command out of the Dockerfiles, where we actually run those specs?

Reply:
That makes sense, I'll remove the ftest builds/tests/scans from this pipeline and follow up with an issue about replacing/removing the ftest dockerfiles.

Reply:
Created an issue for it: https://github.com/elastic/search-developer-productivity/issues/3611